### PR TITLE
Add Barcelona and Friends

### PR DESCRIPTION
**Pull request by mediafinger**
Finally Barcelona is joining Madrid and Malaga and all those German Ruby user groups.

I just registered the domain ruby.barcelona so I can not yet setup the CNAME. Will it work nonetheless, or should I remove the custom domain for now from the PR?

### CODE REVIEW

**Comment by phoet** [comment]
@mediafinger it will work fine through `barcelona.onruby.eu`. just for oauth, it will need the final domain configured, but that can be done afterwards.

File: config/locales/de.label.yml [hunk 1]
```diff
@@ -111,3 +111,10 @@ de:
       :subtitle: Andalucia Ruby User Group
       :meta_desc: Andalucia Ruby User Group
       :custom_recurrence: letzter Dienstag des Monats um 19:30 Uhr
+    barcelona:
+      city: Barcelona
+      name: 'Ruby & Friends'
+      title: 'Ruby & Friends'
+      subtitle: 'vormals: Barcelona on Rails'
+      meta_desc: 'Die Ruby User Group in Barcelona'
+      custom_recurrence: jeden 3. Donnerstag in jedem Monat treffen wir uns um 19:00 Uhr
```
Comment (phoet) [inline config/locales/de.label.yml]: i think this translation is not necessary for a regular schedule. it's used for rules that can not be parsed currently, like "every other month" or so. but to be honest, i don't remember exactly 😄

**Comment by mediafinger** [comment]
❤️

**Comment by phoet** [comment]
🚀 https://barcelona.onruby.eu 🚀

**Comment by phoet** [comment]
oauth is set-up, so once cloudflare dns picks up, everything should be good to go

**Pull request merged**